docs/docs/unit-testing.md

```diff
@@ -217,50 +217,24 @@ by running `npm test -- -u`.
 
 ## Using TypeScript
 
-If you are using TypeScript, you need to make a couple of small changes to your
-config. First install `ts-jest`:
+If you are using TypeScript, you need to make two changes to your
+config.
 
-```shell
-npm install --save-dev ts-jest
+Update the transform in `jest.config.js` to run `jest-preprocess` on files in your project's root directory, where `<rootDir>` is replaced with the correct relative path:
```

Comment:
```suggestion
Update the transform in `jest.config.js` to run `jest-preprocess` on files in your project's root directory, where `<rootDir>` is kept intact as a literal string to dynamically pass the relative path to Jest:
```
Is this accurate to communicate how the `<rootDir>` string is used here? (this was already updated once, based on my own confusion)

Reply:
Yes, I think this is correct. Sorry I meant to update this all week, never got to it. :/